gridt/src/app/about/tutorial/tutorial.page.html

```diff
@@ -0,0 +1,68 @@
+<ion-content>
+  <ion-grid>
+    <!-- <ion-row *ngIf="router.url === '/about'" class="ion-float-right">
+      <ion-col  class="ion-float-right">
+        <ion-button (click)="finish()">
+          <ion-icon name="close" slot="icon-only"></ion-icon>
+        </ion-button>
+      </ion-col>
+    </ion-row> -->
+
+    <ion-row>
+      <ion-col>
+        <ion-buttons *ngIf="router.url === '/about'">
```

Comment:
Why do you only want this button when entering from the about page? Why not always have the option to cancel the tutorial?

Reply:
The idea is that if the tutorial appears on the home page, this is because the user is not subscribed to any movements, so the tutorial fills the home page. The user can always navigate away from the homepage if they want, but the idea is that the user finishes the tutorial and is then navigated to the movements page. If the user opens the tutorial from the about page, this can also happen by accident (selected the wrong button), or because they just want to get a quick look. So for those cases, I put the close button in the corner for easy dismissal.

Reply:
Then you probably also want to overwrite the router return address when coming from the home page. Because now if your home page is empty, and thus the tutorial is shown, what happens when you try and return to the previous page? This is now again the home page, and thus shows you the same tutorial page again. That may give the user the idea that he/she is stuck.

Reply:
I will check it thoroughly tomorrow. The way it is now is that if the
homepage is empty, the tutorial component gets loaded in the homepage. So
the router is still set at 'home', except that the content that is
displayed is from the tutorial. So if the user then navigates back, they
will land at the page where they were before 'home'. But I'll make sure to
check it.

Reply:
You haven't gotten back to him yet.